app/views/devise/mailer/reset_password_instructions.html.slim

```diff
@@ -1,10 +1,12 @@
+/ Subject path for grunt i18nExtract
```

Comment:
Solution to intentionally excluding keys:
- Add a comment with key for subject for each of the emails that has the stealth subject